src/webgl/loading.js

```diff
@@ -322,6 +436,16 @@ function parseObj(model, lines) {
             face[1] !== face[2]
           ) {
             model.faces.push(face);
+            //same material for all vertices in a particular face
+            const materialDiffuseColor =
+             materials[currentMaterial].diffuseColor;
+            for (let i=0 ;i<face.length;i++) {
+              model.vertexColors.push([
```

Comment:
I think elsewhere we treat `vertexColors` as a flat array: https://github.com/processing/p5.js/blob/80338bc715a9baa7d9b4f6fc3de54125a0c280b4/src/webgl/p5.RendererGL.Immediate.js#L117-L122

Maybe we can take out the `[...]` here to be consistent?